uspekhi-gerontologii.csl

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="ru">
+  <info>
+    <title>Uspekhi-gerontologii(numeric, sorted alphabetically, Russian)</title>
+    <id>http://www.zotero.org/styles/uspekhi-gerontologii</id>
+    <link href="http://www.zotero.org/styles/uspekhi-gerontologii" rel="self"/>
+    <link href="http://www.gersociety.ru/information/uspexi/" rel="documentation"/>
+    <author>
+      <name>Valery Mukhin</name>
+      <email>Valery.Mukhin@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
+    <summary>This style is based on the "Russian GOST-2008 style" by Rybakov Dmitriy.</summary>
+    <updated>2016-09-28T10:25:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+ <locale xml:lang="ru">
+    <terms>
+      <term name="et-al"> Рё РґСЂ.</term>
+      <term name="editor">РїРѕРґ СЂРµРґ.</term>
+      <term name="accessed">РґР°С‚Р° РѕР±СЂР°С‰РµРЅРёСЏ</term>
+      <term name="page" form="short">СЃ.</term>
+      <term name="translator">РїРµСЂРµРІРѕРґ</term>
```

Comment:
@adam3smith, don't merge this yet. There are a few more places where the encoding is wrong (I'll take a look later).